.env

```diff
@@ -1,7 +1,5 @@
-VEGA_SIM_VEGA_TAG=443af04ef30fd60086d870eff961916f507c68d6
+VEGA_SIM_VEGA_TAG=5f288d1f1b6ee1cd9f2389e4650ab2994997500e
```

Comment:
I have the following tests failing locally with this tag:

Failing `assertion`:
- test_submit_amend_liquidity (failure makes sense given LP changes)

Failing with `ProposalNotAcceptedError`:
- test_update_market_liquidity_monitoring
- test_update_market_price_monitoring
- test_update_market_instrument
- test_update_market_risk

From the [logs](https://jenkins.ops.vega.xyz/job/common/job/vega-market-sim/1041/console#:~:text=10%3A05%3A21%20%20params%3D%5B%27VEGA_VERSION%27%3A%27443af04ef30fd60086d870eff961916f507c68d6%27%2C%20%27RUN_EXTRA_TESTS%27%3Afalse%2C%20%27RUN_LEARNING%27%3Afalse%2C%20%27ORIGIN_REPO%27%3A%27vegaprotocol/vega%27%2C%20%27TIMEOUT%27%3A%2790%27%2C%20%27VEGA_MARKET_SIM_BRANCH%27%3A%2707546435a834e1a8d538a3829a03753d74dda9c7%27%2C%20%27JENKINS_SHARED_LIB_BRANCH%27%3A%27main%27%5D), looks like the Jenkins tests built with the old tag somehow?